lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -839,7 +840,9 @@ void LocalEnforcer::handle_session_activate_rule_updates(
     const std::string& imsi, SessionState& session,
     const CreateSessionResponse& response,
     std::unordered_set<uint32_t>& charging_credits_received) {
-  RulesToProcess to_activate, to_deactivate;
+  RulesToProcess to_activate, to_deactivate, to_get_bearer;
```

Comment:
shall we change the naming convention to  `pending_flow_activation`, `pending_flow_deactivation`, `pending_bearer_setup`?